crates/router/src/core/payment_methods/cards.rs

```diff
@@ -645,26 +728,45 @@ impl
     not(feature = "payment_methods_v2"),
     not(feature = "customer_v2")
 ))]
+#[allow(clippy::too_many_arguments)]
 pub async fn skip_locker_call_and_migrate_payment_method(
     state: routes::SessionState,
     req: &api::PaymentMethodMigrate,
     merchant_id: id_type::MerchantId,
     key_store: &domain::MerchantKeyStore,
     merchant_account: &domain::MerchantAccount,
     card: api_models::payment_methods::CardDetailFromLocker,
+    should_require_connector_mandate_details: bool,
+    migration_status: &mut RecordMigrationStatusBuilder,
 ) -> errors::RouterResponse<api::PaymentMethodResponse> {
     let db = &*state.store;
     let customer_id = req.customer_id.clone().get_required_value("customer_id")?;
 
-    // In this case, since we do not have valid card details, recurring payments can only be done through connector mandate details.
-    let connector_mandate_details_req = req
-        .connector_mandate_details
-        .clone()
-        .get_required_value("connector mandate details")?;
+    // // In this case, since we do not have valid card details, recurring payments can only be done through connector mandate details.
```

Comment:
```suggestion
    // In this case, since we do not have valid card details, recurring payments can only be done through connector mandate details.
```

Is this comment still valid ?